.scalafmt.conf

```diff
@@ -0,0 +1,24 @@
+#
```

Comment:
Can this file live in dev/? I tried to stuff as many things in there as I could to keep the top-level dir a little less busy.

Reply:
Sure, moved